agent/ansible/pbench/agent/roles/pbench_repo_install/README.md

```diff
@@ -0,0 +1,40 @@
+pbench_repo_install
+==========================
+
+This role is used to install the pbench.repo file in /etc/yum.repos.d
+
+Requirements
+------------
+
+Role Variables
+--------------
+
+Dependencies
+------------
+
+Example Playbook
```

Comment:
Since we are repeating the example playbook so many times, and it has an error in it, it seems like we should put this example playbook into the git repo itself.

Reply:
See above. Fixing it in five places is suboptimal of course, but having people clone the pbench repo just to get one lousy playbook seems excessive. We could link to the file in the repo. Would that be acceptable?

Reply:
A URL to the example playbook in the repo would work fine, especially if we give them a raw link to download the file directly.

Reply:
Done.